### ISSUE TITLE
Membership updates

### ISSUE THREAD

**Issue by orthecreedence** [basisproject/tracker#75]
## Membership / Company

- [ ] Membership updates
  - Membership is a function of a) you live inside of a region's defined geographic area and b) you work at a company that is a member. I can be a member of region A and work at a company in region B. My personal membership does not have to be in the same region as my company's.
  - Personal membership is more about your permanent address. You are a member of the region *you live in* not a member of the region *your company resides in*. If you live in the Minneapolis region but remote work for a California company, you are still a member of the Minneapolis region and have full use of their assets (housing, MoP, bank).
- [ ] Companies have a regional membership. A company has a primary region it is based in, yet has workers who are members of other regions (or the same region). IE, personal membership is still based on being a member of a *member company* but that does not force the personal membership to be in the same region as the company.
  - The bank of the *company's* region would pay workers their credits, as opposed to the region the workers live in paying their credits. The payment for labor comes *from the region the value was created for* which would be the company's.

**Comment by orthecreedence** [comment]
This might change depending on #72.

**Comment by orthecreedence** [comment]
Closing this in favor of #72 and #91.
